src/toil/jobStores/azureJobStore.py

```diff
@@ -48,15 +48,18 @@ def _fetchAzureAccountKey(accountName):
     """
     Find the account key for a given Azure storage account.
 
-    The account key is taken from the AZURE_ACCOUNT_KEY environment
-    variable if it exists, or from looking in the file
-    "~/.toilAzureCredentials". That file has format:
+    The account key is taken from the AZURE_ACCOUNT_KEY_<account> environment
+    variable if it exists, then from plain AZURE_ACCOUNT_KEY, and then from
+    looking in the file "~/.toilAzureCredentials". That file has format:
 
     [AzureStorageCredentials]
     accountName1=ACCOUNTKEY1==
     accountName2=ACCOUNTKEY2==
     """
-    if 'AZURE_ACCOUNT_KEY' in os.environ:
+    
+    if "AZURE_ACCOUNT_KEY_{}".format(accountName) in os.environ:
```

Comment:
I prefer try-except for detecting presence of keys in dictionaries. It is DRY and more efficient. Barring that, I use get() with the 2nd argument.

Also prefer string concatenation for suffixes in favor of of format() or %.

I'll change these things in a continuation PR.